setup.py

```diff
@@ -67,8 +77,6 @@
                  'pep8-naming', 'netCDF4', 'pyproj>=1.9.4,!=2.0.0']
     },
 
-    cmdclass=versioneer.get_cmdclass(),
-
     zip_safe=True,
 
-    download_url='https://github.com/Unidata/MetPy/archive/v{}.tar.gz'.format(ver), )
+    download_url='https://github.com/Unidata/MetPy/archive/v{}.tar.gz'.format(metpy_version))
```

Comment:
This doesn't look right. `metpy_version` is a function above...

Not sure this is totally necessary anyway.

Reply:
If you include the `()` after this, Travis breaks. Any other way to write out this url? Or do we need it? Was just trying to keep what was already in place.